flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/api/ExecutionConfigOptions.java

```diff
@@ -195,4 +195,11 @@
 							"means a kind of disabled operator. Its default value is empty that means no operators are disabled. " +
 							"If the configure's value is \"NestedLoopJoin, ShuffleHashJoin\", NestedLoopJoin and ShuffleHashJoin " +
 							"are disabled. If configure's value is \"HashJoin\", ShuffleHashJoin and BroadcastHashJoin are disabled.");
+
+	public static final ConfigOption<Boolean> SQL_EXEC_SHUFFLE_MODE_ALL_BATCH =
+			key("sql.exec.shuffle-mode.all-batch")
```

Comment:
change to sql.exec.shuffle-mode?
value could be "batch", "pipeline" and so on

Reply:
ok, changed